s3_helper.py

```diff
@@ -1,24 +1,50 @@
+"""
+AWS and AWS S3 Helper functions.
+
+"""
 import boto3
 import botocore.exceptions
 import json
 import logging
 import traceback
 
 
-class S3Helper(object):
+class AWS_helper(object):
+    """
+    Helper class for connecting to AWS.
 
+    """
     def __init__(self, aws_profile=None, logger=False):
+        """
+        Initialization function of the AWS_helper class.
+
+        Parameters:
+            aws_profile: Optional string name of your AWS profile, as set up in
+                the credential file at ~/.aws/credentials. No need to pass in
+                this parameter if you will be using your default profile. For
+                additional information on how to set up the credential file, see
+                https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials_profiles.html
+            logger: Optional parameter. Could pass in a logger object or not pass
+                in anything. If a logger object is passed in, information will be
+                logged instead of printed. If not, information will be printed.
+        """
         self.aws_profile = aws_profile
         self.print_func = print
         if logger:
             self.print_func = logger.info
-
-        self.client = self._get_client()
+        self.session = self._create_aws_session()
 
     def _create_aws_session(self):
+        """
+        Creates AWS session using aws profile name passed in or using aws
+        credentials in environment variables.
+
+        Returns:
+            AWS session object.
+        """
         try:
             if self.aws_profile:
-                session = boto3.session.Session(profile_name=self.aws_profile)
+                session = boto3.session.Session(profile_name=self.aws_profile, region_name='us-east-1')
```

Comment:
Any chance we'll use these functions outside of us-east-1?

Reply:
Removed the default region.